name_topic: "fastech_output"
frequency: 20 #tan so do duoc
get_io_rate: 10 # Hz